CHANGES_NEXT_RELEASE

```diff
@@ -36,6 +36,7 @@
 - Fix: invalid regex patterns detection (#968)
 - Fix: Better error returned on invalid geoquery (Issue #2174)
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
+- Add: q-string filters for compound values of attributes (Issue #1156)
```

Comment:
Fixed in a43b9e2

No, metadatas will come after.
There will be a total of 4 PRs:
1. queries on attribute compound values
2. subscriptions with q-filters on attribute compound values
3. queries on metadata compound values
4. subscriptions with mq-filters on metadata compound values